jena-shex/src/test/files/spec/schemas/1dotCode3fail.shex

```diff
@@ -1,6 +1,11 @@
+PREFIX p: <http://shex.io/extensions/>
```

Comment:
Per my comments elsewhere it would be useful to have a variety of test cases here with variations in whitespace, use of URIs vs prefixed names to identify plugins etc, to check that the parser modifications are sufficiently robust

Although looking at other changes perhaps the reduction in test exclusions is already picking up a selection of pre-existing test cases that were previously skipped?

Reply:
Oops, I was messing with that while I was fiddling with the parser. Restored old state with presumption that the aforementioned [SemAct tests](https://github.com/apache/jena/pull/1637/files#diff-f1b4d58d2a4e81046c490afc8b71496e8440f6aed8e15c3ded996bdb8de0c4c9) provide better coverage.